utils/check_tabs_in_rst.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+$rstFilesWithTabs = (string) shell_exec('git grep -EIPn "\t" -- "*.rst"');
+$rstFilesWithTabs = explode("\n", $rstFilesWithTabs);
+$rstFilesWithTabs = array_map(static function (string $line): array {
+    preg_match('/^(?P<file>[^:]+):(?P<line>[0-9]+):(?P<code>.+)$/', $line, $matches);
+
+    return [
+        'file' => $matches['file'],
+        'line' => $matches['line'],
+        'code' => $matches['code'],
+    ];
+}, array_filter($rstFilesWithTabs));
+
+$normalizedRstFilesWithTabs = [];
+
+foreach ($rstFilesWithTabs as ['file' => $file, 'line' => $line, 'code' => $code]) {
+    if (! isset($normalizedRstFilesWithTabs[$file])) {
+        $normalizedRstFilesWithTabs[$file] = [];
+    }
+
+    $normalizedRstFilesWithTabs[$file][] = ['line' => $line, 'code' => $code];
+}
+
+unset($rstFilesWithTabs);
+
+if ($normalizedRstFilesWithTabs !== []) {
+    printf(
+        "\033[97;41mTabs in RST files were detected:\033[0m\n\n%s\n",
```

Comment:
This string is a little hard to read in color.

And string like `\033[97;41m` is difficult to understand.
Why don't you introduce explaining variables?